pandunia-loge.csv

```diff
@@ -2176,58 +2177,58 @@ lance she||||projectile (missile)|projectile|proyectil (misil)|projétil|сна
 Lau|desha|LA||Laos||Laos||||||||||||||Laos|Laos
-lau||||elder||viejo (anciano)|||||||||||||||starzec
+lau jan||||elder||viejo (anciano)|||||||||||||||starzec
 laurencium|mate|Lr||lawrencium||laurencio|||||||||||||laŭrencio|lawrensium|lorens
-lavi|||zho:老 (lǎo), yue:老 (lau2), wuu:老 (lau3), jpn:老 (rō), vie:lão|old (aged, elderly)|vieux (âgé, ancien)|viejo (anciano)|idoso|пожилой||老 (老年, 年歲大)|年配||già (lão)||||||||stary (sędziwy, wiekowy)
-lavi||||Lao (Laotian)||laosiano||||||||||||||laoslainen|laotański
-layana||||bring|apporter|traer|trazer|приносить||带来|持って来る|가져오다|mang|लाना|আনা|bawa|kuleta|getirmek|alporti (venigi)|tuoda|
+lau|||zho:老 (lǎo), yue:老 (lau2), wuu:老 (lau3), jpn:老 (rō), vie:lão|old (aged, elderly)|vieux (âgé, ancien)|viejo (anciano)|idoso|пожилой||老 (老年, 年歲大)|年配||già (lão)||||||||stary (sędziwy, wiekowy)
+lau di||||Lao (Laotian)||laosiano||||||||||||||laoslainen|laotański
+fa lai||||bring|apporter|traer|trazer|приносить||带来|持って来る|가져오다|mang|लाना|আনা|bawa|kuleta|getirmek|alporti (venigi)|tuoda|
 lazanya|||fra:lasagne, eng:lasagna, spa:lasaña, por:lasanha, rus:лазанья (lazanya), ara:(lazanyā), jpn:ラザニア (razania), kor:라자냐 (lajanya), hin:लज़ैन्या (lazenyā)|lasagna|lasagne|lasaña|lasanha|лазанья|لَازَانْيَا|千层面|ラザニア|라자냐||लज़ैन्या||||lazanya||lasagne|lazania
-lazuri||||azure|azur|azur (blao)|azur (blau)|лазурь|||||||||||||
-lazurpetre|||fas:لاجورد (lājevard), eng:lazuli, spa:lapislázuli, rus:лазурит (lazurit), ara:لازورد (lāzaward), hin:लाजवर्द (lājavarda), jpn:ラピスラズリ (rapisurazuri), may:lazuardi, tur:lazur taşı|lapis lazuli|lapis-lazuli|lapislázuli|lápis-lazúli|лазурит|لَازَوَرْد|青金岩|ラピスラズリ (瑠璃)|||लाजवर्द (वैडूर्य)||||lazur taşı||lapislatsuli|lapis lazuli
+lazur|rang|||azure|azur|azur (blao)|azur (blau)|лазурь|||||||||||||
+lazur petra|||fas:لاجورد (lājevard), eng:lazuli, spa:lapislázuli, rus:лазурит (lazurit), ara:لازورد (lāzaward), hin:लाजवर्द (lājavarda), jpn:ラピスラズリ (rapisurazuri), may:lazuardi, tur:lazur taşı|lapis lazuli|lapis-lazuli|lapislázuli|lápis-lazúli|лазурит|لَازَوَرْد|青金岩|ラピスラズリ (瑠璃)|||लाजवर्द (वैडूर्य)||||lazur taşı||lapislatsuli|lapis lazuli
 lefte|||eng:left, rus:левый (levyy)|left||lado izquerdo|esquerdo|левый||左|左|||बायीं||kiri|kushoto|sol||vasen|lewa strona
-lefti||||left hand||izquierdo|||||||||||||||lewy (lewostronny, z lewej strony)
-leftiste||||leftist (left-winger)||izquerdista||||||||||||||vasemmistolainen|lewicowiec (lewak)
-legi|||eng:light, spa:ligero, fra:léger, rus:лёгкий (l’ogkiy), pol:lekki, ben:লঘু (laghu)|light (weightless)|léger|ligero|||||||||||||||lekki
+lefte di||||left hand||izquierdo|||||||||||||||lewy (lewostronny, z lewej strony)
+lefte jui ja||||leftist (left-winger)||izquerdista||||||||||||||vasemmistolainen|lewicowiec (lewak)
+lege|||eng:light, spa:ligero, fra:léger, rus:лёгкий (l’ogkiy), pol:lekki, ben:লঘু (laghu)|light (weightless)|léger|ligero|||||||||||||||lekki
```

Comment:
Are you sure about the Bangla word? I found out that "laghu" has a general diminutive meaning Sanskrit and Hindi meaning easy, small, short, insignificant, etc. See https://en.wiktionary.org/wiki/%E0%A4%B2%E0%A4%98%E0%A5%81#Sanskrit

On another thought, a versatile word like that could be useful! Then we could add Hindi to the list of sources and change the word to "ligu" by combining the first vowel of eng:light, spa:ligero, por:ligeiro and the last vowel of hin:laghu, ben:laghu.

Then light-weight would be **ligu veze di** but you could say simply **ligu** when the context permits.

Maybe the meaning of heavy (**bari**) could be extended in a similar manner, though in Sanskrit the versatile word is "guru". See https://en.wiktionary.org/wiki/%E0%A4%97%E0%A5%81%E0%A4%B0%E0%A5%81#Sanskrit

Reply:
I don't remember, but I mite have gotten some of these meanings a bit rong, since "light" is so ambiguous in English, and not all diccionaries are thoro enuff to disambiguate things like that.

the general use word sounds good, tho!  I think it makes sense to extend **bari** to mean "big" or "important", since **guru** already means "guru".